braveautogen.py

```diff
@@ -105,20 +102,7 @@
 if args.test is None:
   repo.create_issue(title=mactitle,body=bigline,assignee="LaurenWags",milestone=milestone_dictionary[key] ,labels=maclist)
 
-print("Win32 Checklist:")
-bigline = "## Per release specialty tests\n"
-for line in win32_checklist:
-  bigline += line + "\n"
-bigline = bigline + template
-print(bigline)
-print("")
-wintitle = "Manual test run on Windows ia-32 for " + key
-winlist = ['OS/Windows', 'release-notes/exclude', 'tests']
-
-if args.test is None:
-  repo.create_issue(title=wintitle,body=bigline,assignee="luixxiul",milestone=milestone_dictionary[key] ,labels=winlist)
-
-print("Win64 Checklist:")
+print("Winx64 Checklist:")
```

Comment:
all the other scripts are using `Win64 Checklist:` but this was changed to `Winx64 Checklist:`